src/main/java/io/buildpacks/example/sample/CalculatorService.java

```diff
@@ -23,4 +23,35 @@ public int divide(int a, int b) throws IllegalArgumentException {
         }
         return a / b;
     }
```

Comment:
3. **Exception Handling**:
    - In the `divide(int a, int b)` method, the exception handling is incorrect. Instead of catching `ArithmeticException`, the method should declare that it `throws IllegalArgumentException` in the method signature like in the other `divide` method.